osf/metadata/utils.py

```diff
@@ -0,0 +1,86 @@
+from website import settings
+
+
+SUBJECT_SCHEME = 'bepress Digital Commons Three-Tiered Taxonomy'
+
+DATACITE_RESOURCE_TYPE_MAP = {
+    'Audio/Video': 'Audiovisual',
+    'Dataset': 'Dataset',
+    'Image': 'Image',
+    'Model': 'Model',
+    'Software': 'Software',
+    'Book': 'Text',
+    'Funding Submission': 'Text',
+    'Journal Article': 'Text',
+    'Lesson': 'Text',
+    'Poster': 'Text',
+    'Preprint': 'Text',
+    'Presentation': 'Text',
+    'Research Tool': 'Text',
+    'Thesis': 'Text',
+    'Other': 'Text',
+    '(:unas)': 'Other'
```

Comment:
Nit: Things like a "Poster" or "Research Tool" or "Presentation" aren't necessarily "Text" if there's any possibility it couldn't be completely formatted as a text, we should play it safe and be go with "Other"

Reply:
This was a product decision, see [the file metadata assessment
doc](https://docs.google.com/spreadsheets/d/1A3q6rbD3cE3dxzAsXQRgiadfvIu8fU3hPyg7-YFNPjY/) in the "OSF Resource Types" tab.